pkg/convert/json_ng.go

```diff
@@ -0,0 +1,61 @@
+package convert
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+func FromNgJson(b []byte) (model.Messages, error) {
+	incoming := make(map[string]interface{})
```

Comment:
You can simplify a lot. We know that ng translation file contains concrete schema. We can define its type in go to simplify serialisation to/from JSON.

```go
type NgJson struct {
  Locale       language.Tag `json:"locale"`
  Translations map[string]string `json:"translations"`
}
```

Reply:
You said that I have to use lower case fields in struct but the fields are not visible to json.Marshal because it is not part of the package.
So I used upper-case fields